templates/firefox-ios.xliff

```diff
@@ -2796,6 +2827,14 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="Intro.Slides.Automatic.Privacy.Description">
+        <source>Privacy by default blocks malware and stops trackers.</source>
```

Comment:
I don't understand this sentence. What's the subject here? Is it "Privacy" or "Privacy by default"?

Reply:
> I don't understand this sentence. What's the subject here? Is it "Privacy" or "Privacy by default"?

This is the description for using a browser that respects privacy of their users at all times as in Privacy by default.

Ref UI - 
https://app.abstract.com/share/fb3066fc-2401-48a1-951d-609eb81c3913?collectionId=912f4b7f-895d-40ec-b7c9-0a2c9cd5af8e&collectionLayerId=e965dff4-deb4-47fd-96dc-b10cc050233c

Reply:
It's still unclear to me what "Privacy by default" stands for: should it be intended as a sort of feature name, does it refer to the default privacy setting? Right now it's really hard to translate.

@betsymi
Any chance you could shed some light?

Reply:
Not a feature name, no. It refers to our default privacy settings. 

If this is too confusing to translate, we can adjust to `Enhanced Tracking Protection blocks malware and stops trackers.`

Reply:
I think that would help translators a lot.